spec/migrations/20190226184255_check_guid_uniqueness_spec.rb

```diff
@@ -0,0 +1,57 @@
+require_migration
+
+describe CheckGuidUniqueness do
+  let(:automate_workspace) { migration_stub(:AutomateWorkspace) }
+  let(:ext_management_system) { migration_stub(:ExtManagementSystem) }
+  let(:host) { migration_stub(:Host) }
+  let(:job) { migration_stub(:Job) }
+  let(:lifecycle_event) { migration_stub(:LifecycleEvent) }
+  let(:miq_action) { migration_stub(:MiqAction) }
+  let(:miq_ae_workspace) { migration_stub(:MiqAeWorkspace) }
+  let(:miq_event_definition) { migration_stub(:MiqEventDefinition) }
+  let(:miq_policy) { migration_stub(:MiqPolicy) }
+  let(:miq_region) { migration_stub(:MiqRegion) }
+  let(:miq_server) { migration_stub(:MiqServer) }
+  let(:miq_set) { migration_stub(:MiqSet) }
+  let(:miq_widget) { migration_stub(:MiqWidget) }
+  let(:miq_worker) { migration_stub(:MiqWorker) }
+  let(:scan_item) { migration_stub(:ScanItem) }
+  let(:service) { migration_stub(:Service) }
+  let(:service_template) { migration_stub(:ServiceTemplate) }
+  let(:vm) { migration_stub(:Vm) }
+
+  migration_context :up do
+    it "dup resets guid" do
+      [ext_management_system,
+       host,
+       job,
+       lifecycle_event,
+       miq_action,
+       miq_ae_workspace,
+       miq_event_definition,
+       miq_policy,
+       miq_region,
+       miq_server,
+       miq_widget,
+       miq_worker,
+       scan_item,
+       service,
+       service_template,
+       miq_set,
+       vm].each do |obj|
+        p obj.to_s
```

Comment:
Please remove debugging statements

Reply:
I left this in because it's nice to see what object it's testing, is there a way of doing this that is better?

Reply:
Yes, the `[].each` can be moved up one level of nesting.

```ruby
migration_context :up do
  [things...].each do |test_subject|
    it "#{thing.name} guids are unique" do
      ...the test logic...
    end
  end
end
```